src/Helpers/Scripts_Helper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace JustCoded\WP\Composer\Helpers;
+
+
+use Composer\IO\IOInterface;
+
+/**
+ * Class Scripts_Helper
+ *
+ * @package JustCoded\WP\Composer\Helpers
+ */
+class Scripts_Helper {
+
+	/**
+	 * Help to parse arguments to key/value
+	 *
+	 * @param array $arguments
+	 *
+	 * @return array
+	 */
+	public static function parse_arguments( $arguments = array() ) {
+		$args_ready = array();
+		foreach ( $arguments as $key => $arg_str ) {
+			preg_match( '/(^-[^=? ?]*)(.)/', $arg_str, $matches );
+			if ( empty( $matches ) ) {
+				$args_ready[] = $arg_str;
+			} else {
+				if ( '-' !== $matches[1] ) {
+					$matches[1] = str_replace( '-', '', $matches[1] );
+					$args_ready[$matches[1]] = explode( $matches[2], $arg_str )[1];
+				}
+			}
```

Comment:
need to debug (will do once all other comments ready).
usually if you separate by space - this will be different args, so you need to check next array element.

regular expression - space character has special char, you can't use usual spaces inside regular expressions

Reply:
this variant works with spaces and =. 
http://i.prntscr.com/ykx4e_I_QW_36_DI8JU8gA.png

space character - has been placed in regular expression.  \s